pydantic_ai_slim/pydantic_ai/tools.py

```diff
@@ -503,6 +509,18 @@ class ToolDefinition:
     For MCP tools, this contains the `meta`, `annotations`, and `output_schema` fields from the tool definition.
     """
 
+    input_examples: list[dict[str, Any]] | None = None
```

Comment:
Anthropic calls them `input_examples` because they have `input_schema`, but we have `parameters_json_schema`, so let's call it either `parameters_examples` or, better, just `examples`.